theme/connector/view/_Learn.scss

```diff
@@ -941,3 +941,33 @@ img.detail-has-data-very-small {
     text-transform: uppercase;
   }
 }
+
+/* Data export button, with light arrow icon */
+.x-btn-rounded-inverted-accent-small .x-btn-arrow-light {
+  background-image: url(data:image/gif;base64,R0lGODlhCgAKAHAAACH5BAEAAAIALAAAAAAKAAoAgQAAAP///wAAAAAAAAIMlI+pyxr/XFyh2ZsKADs=);
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+}
+
+/* Data export menu item, with download icon */
+.export-item {
+  background-color: $white;
+
+  .x-menu-item-text {
+    margin-left: 10px;
+  }
+  .export-item-icon {
+    background-image: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDIzLjAuNiwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IgoJIHZpZXdCb3g9IjAgMCA5IDEwIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA5IDEwOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+CjxzdHlsZSB0eXBlPSJ0ZXh0L2NzcyI+Cgkuc3Qwe2ZpbGw6IzAwMDAwMDt9Cjwvc3R5bGU+CjxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik05LDd2MnYxSDhIMUgwVjdoMXYyaDdWN0g5eiBNNy4zMywzLjk0TDYuNjIsMy4yM0w1LDQuODVWMEg0djQuODVMMi4zOCwzLjIzTDEuNjcsMy45NEw0LjUsNi43N0w3LjMzLDMuOTR6Ii8+Cjwvc3ZnPgo=);
```

Comment:
Can those svgs be saved in webapp/Connector/images/ as files and then used as file path?

Reply:
yes, but I thought it would be easier to use just the inlined data versions rather than check in two separate files that are only used in one place each. Do you know if there are pros/cons of one over the other?

Reply:
swtiched to the more traditional SVG files...

Reply:
It's more of a personal preference. I found the file way to be more readable and consistent. For the "export" icon case, the inline method might have the advantage of being able to define a single css selector with the svg, but use additional selector to change just the color part of it. It looks good as is.